### PR TITLE
Bug fixes for international Degrees & GCSEs

### PR DESCRIPTION
**Pull request by davidgisbey**
## Context

There are a few bugs at the moment in the international degrees section. This PR tackles two separate cards and two additional bugs that I noticed while fixing the first two.

## Changes proposed in this pull request

Both the degree and GCSE review component

1. The review components for international degrees/GCSEs should have a row showing stating if they have a NARIC statement. If no it should not  show the naric_reference or comparable UK qualification rows.

Degree review page  

NARIC ref provided 

image

NARIC ref not provided

image

GCSE review page 

NARIC ref provided

image

NARIC ref not provided 

image

Degrees - NARIC page

Degree specific bugs

1. Prefill the form correctly so the 'No' radio button works correctly

Before 

image

After

image

2. If a candidate has added a NARIC reference number. Then chooses to change their response to 'No' it sets the naric_reference and comparable_uk_degree to nil (previously it retained their values)
 
## Guidance to review

Test it locally/on the review app 👍 

## Link to Trello card

https://trello.com/c/0QxX0k6r/2052-%F0%9F%8C%90-comparable-uk-degree-shown-in-summary-card-for-international-degree-if-answered-no-to-having-a-naric-statement
https://trello.com/c/ojQCw0IY/2056-%F0%9F%8C%90-empty-naric-ref-field-in-summary-when-no-naric-equivalent-for-degree

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

**Comment by paulrobertlloyd** [comment]
Not quite right on the review page, need to show either 3 lines relating to NARIC, or one:

No NARIC statement:

Screenshot 2020-09-22 at 15 59 56

Has NARIC statement:

Screenshot 2020-09-22 at 16 01 14

Also, I think we should be consistent and say `NARIC`, not `UK NARIC`. Do you agree @EmmaFrith?

**Comment by EmmaFrith** [comment]
> Also, I think we should be consistent and say `NARIC`, not `UK NARIC`. Do you agree @EmmaFrith?

yup agree @paulrobertlloyd

**Review by tvararu — APPROVED** [review]

File: app/components/candidate_interface/gcse_qualification_review_component.rb [hunk 1]
```diff
@@ -94,19 +89,38 @@ def country_row
       }
     end
 
-    def naric_row
+    def naric_statment_row
+      return nil unless FeatureFlag.active?('international_gcses') && application_qualification.qualification_type == 'non_uk'
+
+      {
+        key: 'Do you have a NARIC statment of comparability?',
```
Comment (paulrobertlloyd) [inline app/components/candidate_interface/gcse_qualification_review_component.rb:96]:
```suggestion
        key: 'Do you have a NARIC statement of comparability?',
```

Sorry, this was my spelling mistake!

File: config/locales/application_form.yml [hunk 2]
```diff
@@ -190,10 +190,14 @@ en:
         button: Add another degree
       delete: Delete degree
       confirm_delete: Yes I’m sure - delete this degree
+      naric_statment:
+        label: NARIC statment
+        review_label: Do you have a NARIC statment of comparability?
```
Comment (paulrobertlloyd) [inline config/locales/application_form.yml:195]:
```suggestion
        review_label: Do you have a NARIC statement of comparability?
```

Ditto.

**Review by paulrobertlloyd — CHANGES_REQUESTED** [review]
Just a typo to fix, then 🚢

**Pull request merged**